test/integration/TokenRejectIntegrationTest.js

```diff
@@ -0,0 +1,834 @@
+import { expect } from "chai";
+import {
+    AccountAllowanceApproveTransaction,
+    AccountBalanceQuery,
+    AccountCreateTransaction,
+    AccountUpdateTransaction,
+    Hbar,
+    NftId,
+    PrivateKey,
+    TokenCreateTransaction,
+    TokenMintTransaction,
+    TokenPauseTransaction,
+    TokenRejectTransaction,
+    TokenType,
+    TransactionId,
+    TransferTransaction,
+} from "../../src/exports.js";
+import IntegrationTestEnv from "./client/NodeIntegrationTestEnv.js";
+
+describe("TokenRejectIntegrationTest", function () {
+    let env, tokenId, receiverId, receiverPrivateKey;
+
+    beforeEach(async function () {
+        env = await IntegrationTestEnv.new();
+
+        // create token
+        const tokenCreateTx = await new TokenCreateTransaction()
+            .setTokenName("ffff")
+            .setTokenSymbol("F")
+            .setDecimals(3)
+            .setInitialSupply(1000000)
+            .setTreasuryAccountId(env.operatorId)
+            .setPauseKey(env.operatorKey)
+            .setAdminKey(env.operatorKey)
+            .setSupplyKey(env.operatorKey)
+            .execute(env.client);
+
+        tokenId = (await tokenCreateTx.getReceipt(env.client)).tokenId;
+
+        // create receiver account
+        receiverPrivateKey = await PrivateKey.generateECDSA();
+        const receiverCreateAccount = await new AccountCreateTransaction()
+            .setKey(receiverPrivateKey)
+            .setInitialBalance(new Hbar(1))
+            .setMaxAutomaticTokenAssociations(-1)
+            .execute(env.client);
+
+        receiverId = (await receiverCreateAccount.getReceipt(env.client))
+            .accountId;
+    });
+
+    describe("Fungible Tokens", function () {
+        it("should execute TokenReject Tx", async function () {
+            this.timeout(120000);
+
+            // create another token
+            const tokenCreateTx2 = await new TokenCreateTransaction()
+                .setTokenName("ffff2")
+                .setTokenSymbol("F2")
+                .setDecimals(3)
+                .setInitialSupply(1000000)
+                .setTreasuryAccountId(env.operatorId)
+                .setAdminKey(env.operatorKey)
+                .setSupplyKey(env.operatorKey)
+                .execute(env.client);
+
+            const { tokenId: tokenId2 } = await tokenCreateTx2.getReceipt(
```

Comment:
nit: Other place you did:
`let tokenId2 = (await tokenCreateTx2.getReceipt(env.client)).tokenId;` You can stick to one approach.

Reply:
Depends on the use case. If I can initialize the variable I use destructure assignment but after the variable/constant is initialized I can't use destructuring. Just JS stuff 🚩.

Reply:
This isn't an important notice just would say to stick to one approach.